minio/m3
# Add Prometheus support for monitoring MinIO ports

To Test locally:

- Deploy M3 as usual, add a tenant.
- `kubectl get pods`
- Get the Pod name of Prometheus.
- Use `kubectl port-forward <prometheus-pod-name> 9090:9090`
- Open up localhost:9090, you should be able to see Prometheus dashboard with MinIO tenant containers as targets.
- To test auto discovery, add another tenant.
- Prometheus dashboard should automatically update with new MinIO tenant containers
